ocfl-java-core/src/main/java/edu/wisc/library/ocfl/core/extension/storage/layout/config/NTupleOmitPrefixStorageLayoutConfig.java

```diff
@@ -0,0 +1,180 @@
+package edu.wisc.library.ocfl.core.extension.storage.layout.config;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import org.apache.commons.lang3.StringUtils;
+
+import edu.wisc.library.ocfl.core.extension.storage.layout.NTupleOmitPrefixStorageLayoutExtension;
+import edu.wisc.library.ocfl.core.extension.OcflExtensionConfig;
+
+/**
+ * Configuration for the <a href="https://ocfl.github.io/extensions/0007-n-tuple-omit-prefix-storage-layout.md">
+ * N Tuple Storage Layout</a> extension.
+ *
+ * @author vcrema
+ * @since 2021-10-25
+ */
+public class NTupleOmitPrefixStorageLayoutConfig implements OcflExtensionConfig {
+
+	public static final String ZERO_PADDING_LEFT = "left";
+	public static final String ZERO_PADDING_RIGHT = "right";
+	public static final String ZERO_PADDING_NONE = "none";
+
+	private String delimiter;
+	private int tupleSize = 3;
+	private int numberOfTuples = 3;
+	private String zeroPadding = ZERO_PADDING_LEFT;
+	private boolean reverseObjectRoot = false;
+
+	@JsonIgnore
+	@Override
+	public String getExtensionName() {
+		return NTupleOmitPrefixStorageLayoutExtension.EXTENSION_NAME;
+	}
+
+	@JsonIgnore
+	@Override
+	public boolean hasParameters() {
+		return true;
+	}
+
+	/**
+	 * @return the delimiter marking end of prefix
+	 */
+	public String getDelimiter() {
+		return delimiter;
+	}
+
+	/**
+	 * @return tupleSize - the segment size (in characters) to split the digest
+	 *         into
+	 */
+	public int getTupleSize() {
+		return tupleSize;
+	}
+
+	/**
+	 * @return The number of segments to use for path generation
+	 */
+	public int getNumberOfTuples() {
+		return numberOfTuples;
+	}
+
+	/**
+	 * return zeroPadding - Indicates whether to use left or right zero padding
+	 * for ids less than tupleSize * numberOfTuples
+	 */
+	public String getZeroPadding() {
+		return zeroPadding;
+	}
+
+	/**
+	 * return true or false, indicates that the prefix-omitted, padded object
+	 * identifier should be reversed
+	 */
+	public boolean reverseObjectRoot() {
+		return reverseObjectRoot;
+	}
+
+	/**
+	 * The case-insensitive, delimiter marking the end of the OCFL object
```

Comment:
Ack! I hate that I missed that this extension defines the delimiter as case-insensitive too. It makes the extension needlessly complex.